archive.sh

```
@@ -0,0 +1,110 @@
+#!/bin/bash
```

rename this scirpt properly
r360-archive.sh

next time we will have alot of scripts here

also, you haven't upload pb-archive.sh

---

will be doing it now, i honestly forgot about it 😓😓😓

---

PB CHANGES BELOW:

-Folder Versioning Logic
Prevents accidental overwrites by automatically renaming destination folders (e.g., _updated, _updated_v2) if a folder with the same name already exists.

-move_item Function (DRY Principle)
Consolidates repetitive mv and mkdir operations into a single function. This reduces code duplication and enforces consistent checks and behavior for all file moves.

-Folder Existence and Emptiness Checks
Ensures that each source folder exists and contains files before attempting to move it. This prevents silent failures and protects against moving empty or non-existent folders.

-Archive Logging
Automatically generates an archive_log.txt in the destination folder, documenting the event ID, timestamp, target paths, and what was moved. Useful for tracking and accountability.

-User Confirmation and Input Validation
Implements a clear case structure for confirming user input before proceeding. It ensures the event ID is reviewed and prevents accidental execution with incorrect or blank inputs.